arrow-fx/src/test/kotlin/arrow/fx/ResourceTest.kt

```diff
@@ -59,16 +55,18 @@ class ResourceTest : UnitSpec() {
 private fun Resource.Companion.eqK() = object : EqK<ResourcePartialOf<ForIO, Throwable>> {
   override fun <A> Kind<ResourcePartialOf<ForIO, Throwable>, A>.eqK(other: Kind<ResourcePartialOf<ForIO, Throwable>, A>, EQ: Eq<A>): Boolean =
     (this.fix() to other.fix()).let {
-      val ls = it.first.invoke { IO.just(1) }.fix()
-      val rs = it.second.invoke { IO.just(1) }.fix()
+      val ls = it.first.use { IO.just(it) }.fix().attempt()
+      val rs = it.second.use { IO.just(it) }.fix().attempt()
       val compare = IO.applicative().mapN(ls, rs) { (l, r) -> l == r }.fix()
 
-      compare.unsafeRunTimed(5.seconds) == Some(true)
+      compare.unsafeRunTimed(10.seconds) == Some(true)
```

Comment:
Are the test running so much slower?

We shouldn't use `unsafeRunTimed`, it's not a safe alternative for timing out. It limits the time any `async` boundary can take, so infinite `flatMap`s never time-out. We should forward to `IO.eq` here instead.

Reply:
Not really, just forgot to remove this after testing :D